src/azul/project/hca/transformers.py

```diff
@@ -502,6 +502,10 @@ def _parse_zarr_file_name(file_name):
     return zarr_name, sub_name
 
 
+def _replace_path_delimiter(file_name: str) -> str:
```

Comment:
No need to extract a function here. It doesn't add clarity and has only one call site.